src/core/types.ts

```diff
@@ -11,6 +11,7 @@ export interface Pkg {
 export interface PkgInfo {
   name: string;
   version?: string;
+  labels?: object;
```

Comment:
please consider the narrowest possible type for now - otherwise we'll be forced to consider how complex objects get de/serialised. `Date`s will not survive the round trip, for example, and will lead to bugs. Do we want/need to support nested objects yet? Do we have to decide right now? If not, this might be good enough for now:

```typescript
  labels?: {
    [key: string]: string | number | boolean;
  };
```

we can always expand it later (a minor version bump), but restricting it later will be a major breaking change to the schema.